cap/modules/services/views/cern.py

```diff
@@ -25,15 +25,15 @@
 
 """CAP CERN service views."""
 
+import json
 import ldap
-import requests
-
-from flask import Blueprint, current_app, jsonify, request
-from invenio_files_rest.models import FileInstance, ObjectVersion
+from flask import jsonify, request, current_app, make_response
 
+from . import blueprint
 from cap.modules.access.utils import login_required
 
-from . import blueprint
+
+LDAP_SERVER_URL = 'ldap://xldap.cern.ch'
```

Comment:
I know that Pamfilos may not agree, but i would move it to the config file with all the rest external APIs urls (like Illias suggested before), so everything in one place